source/includes/_instances.md

```diff
@@ -149,10 +151,13 @@ data/attributes/test-id* | Test id (not display id) | true |
 data/attributes/planned-execution | date field of planned-execution | false |
 data/attributes/version | string of instance version | false |
 data/attributes/priority | string of instance priority | false |
-data/attributes/assigned-to-id | user assigned-to id (not Display ID) - [users list](#users)  | false |
+data/attributes/assigned-to-id | user or group assigned-to id (not Display ID) - [users list](#users)  | false |
+data/attributes/assigned-to-type | assigned-to type (user or group) | false |
```

Comment:
Should show that thee default is User

Reply:
is this sentence (from the bullet under the parameters table) not enough?
>> If only assigned-to-id is passed, then type will be considered as user.